botfront/imports/ui/components/nlu/models/NLUModel.jsx

```diff
@@ -165,35 +164,44 @@ class NLUModel extends React.Component {
         if (instance) {
             tabs.push({
                 menuItem: 'API',
-                render: () => (
-                    <React.Suspense fallback={<Loader active />}>
-                        <API model={model} instance={instance} />
-                    </React.Suspense>
-                ),
+                render: () => (<API model={model} instance={instance} />),
```

Comment:
Why did you remove the React Suspense here ?

Reply:
App was crashing because of some React.lazy not working properly. Nathan told to leave it for now and we will fix it later. So I removed it.